fieldservice_recurring/security/recurring_security.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+
+    <record id="rule_fsm_frequency_multi_company" model="ir.rule">
+        <field name="name">FSM Frequency multi-company</field>
+        <field name="model_id" ref="model_fsm_frequency"/>
+        <field name="global" eval="True"/>
+        <field name="domain_force">['|', ('company_id', '=', False), ('company_id', 'child_of', [user.company_id.id])]</field>
```

Comment:
HI @kittiu, thanks very much for your contribution. 
I have this bug on fms_frequency. 
Traceback (most recent call last):
  File "/odoo/src/odoo/http.py", line 656, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/odoo/src/odoo/http.py", line 314, in _handle_exception
    raise pycompat.reraise(type(exception), exception, sys.exc_info()[2])
  File "/odoo/src/odoo/tools/pycompat.py", line 87, in reraise
    raise value
  File "/odoo/src/odoo/http.py", line 698, in dispatch
    result = self._call_function(**self.params)
  File "/odoo/src/odoo/http.py", line 346, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/odoo/src/odoo/service/model.py", line 97, in wrapper
    return f(dbname, *args, **kwargs)
  File "/odoo/src/odoo/http.py", line 339, in checked_call
    result = self.endpoint(*a, **kw)
  File "/odoo/src/odoo/http.py", line 941, in __call__
    return self.method(*args, **kw)
  File "/odoo/src/odoo/http.py", line 519, in response_wrap
    response = f(*args, **kw)
  File "/odoo/src/addons/web/controllers/main.py", line 904, in search_read
    return self.do_search_read(model, fields, offset, limit, domain, sort)
  File "/odoo/src/addons/web/controllers/main.py", line 926, in do_search_read
    offset=offset or 0, limit=limit or False, order=sort or False)
  File "/odoo/src/odoo/models.py", line 4572, in search_read
    records = self.search(domain or [], offset=offset, limit=limit, order=order)
  File "/odoo/src/odoo/models.py", line 1561, in search
    res = self._search(args, offset=offset, limit=limit, order=order, count=count)
  File "/odoo/src/odoo/models.py", line 4105, in _search
    self._apply_ir_rules(query, 'read')
  File "/odoo/src/odoo/models.py", line 3957, in _apply_ir_rules
    where_clause, where_params, tables = Rule.domain_get(self._name, mode)
  File "/odoo/src/odoo/addons/base/models/ir_rule.py", line 126, in domain_get
    query = self.env[model_name].sudo()._where_calc(dom, active_test=False)
  File "/odoo/src/odoo/models.py", line 3896, in _where_calc
    e = expression.expression(domain, self)
  File "/odoo/src/odoo/osv/expression.py", line 672, in __init__
    self.parse()
  File "/odoo/src/odoo/osv/expression.py", line 853, in parse
    raise ValueError("Invalid field %r in leaf %r" % (left, str(leaf)))
 ValueError: Invalid field 'company_id' in leaf "<osv.ExtendedLeaf: ('company_id', '=', False) on fsm_frequency (ctx: )>"